engine/DEV.md

```diff
@@ -42,6 +42,12 @@ To show the file nicely formated, use: `dune describe pp lib/types.ml` (or `dune
 
 You can also use `dune utop` and then `#show_type Hax_engine.Types.SOME_TYPE` and `#show_constructor Hax_engine.Types.SOME_CONSTRUCTOR`.
 
+### Visitors
+The module `Ast_visitors` provides a `Make` functor that takes a feature set and outputs a module that defines the `map`, `mapreduce` and `reduce` classes.
+
+Those visitors are generated by `./utils/generate_visitors`. 
+To see the implementation of the `Ast_visitors` module, run `dune describe pp lib/ast_visitors.ml`.
```

Comment:
Running this I get
```
File "utils/generate_visitors/errors.ml", line 27, characters 29-35:
27 |     | UnsupportedCoreType of string
                                  ^^^^^^
Error: Unbound value string_of_yojson
```

Reply:
Thanks for the feedback, that's odd. Might be a library version or something, but anyway, I just pushed 3887a9f, can you check if that's fixing it?

Reply:
Also, do you get this error when building regularly?

Reply:
Yes, this fails for the regular build as well.

Reply:
Ok! But the patch makes things work, right?